app/helpers/chat_migration_message.rb

```diff
@@ -0,0 +1,24 @@
+module ChatMigrationMessage
+
+  DISPLAY_MESSAGE_YEAR  = 2016
+  DISPLAY_MESSAGE_MONTH = 2
+  DISPLAY_MESSAGE_DAY   = 6
+
+  class Message
+    def initialize
+      @migration_date = Date.new(DISPLAY_MESSAGE_YEAR, DISPLAY_MESSAGE_MONTH, DISPLAY_MESSAGE_DAY)
+    end
+
+    def display_message?
+      Date.today == @migration_date
+    end
+  end
+
+  def display_chat_migration_message?
+    ChatMigrationMessage::Message.new.display_message?
+  end
+
+  def alternate_hours_active?
+    display_chat_migration_message?
+  end
+end
```

Comment:
It looks like `ChatMigrationMessage::Message` is only being used within :display_chat_migration_message?, could it be collapsed/inlined? eg:

```
module ChatMigrationMessage
  MIGRATAION_DATE = Date.new(2016, 2, 6)

  def display_chat_migration_message?
    Date.today == MIGRATAION_DATE
  end

  def alternate_hours_active?
    display_chat_migration_message?
  end
end
```